### ISSUE TITLE
More verbose bench tags

### ISSUE THREAD

**Issue by awf** [Mikolaj/mostly-harmless#10]
https://github.com/Mikolaj/mostly-harmless/blob/c27ec3b0e04f0e376adf00036beb4bd6f074a802/bench/ProdMostlyHarmlessTools.hs#L17

As we look at more benchmarks, it might be helpful to have these strings be descriptive across the repo, e.g. replace with
```Haskell
 bgroup "prod 100"
```
or even, if we also benchmark functions in other subdirs,
```Haskell
 bgroup "bench/prod 100"
```

**Comment by Mikolaj** [comment]
Good idea. However, I'm already hitting lenght limit in the html reports. Are they useful? Do we care that they have such strings truncated? Could we have a convention that the `prod`/`mnist` distinction is conveyed by .html file name and the name of the discussion subforum the bechmark is placed in, and so can be omitted here?

We only benchmark things contained in benchmark directories. However, each library has its own benchmark directory 'LIBRARYNAME-bench`. If that's helpful I can mark the benchmark by that directory name, not by the library name (especially that our library is nameless).
